themes/agnoster/agnoster.theme.sh

```diff
@@ -296,6 +297,39 @@ function prompt_virtualenv {
   fi
 }
 
+### pyenv prompt
+function prompt_pyenv {
+  if [[ $PYENV_VIRTUALENV_INIT == 1 ]]; then
+    # Priority is shell > local > global
+    local PYENV_SHELL=$(pyenv shell 2>&1)
+    local PYENV_LOCAL=$(pyenv local 2>&1)
+    local PYENV_GLOBAL=$(pyenv global 2>&1)
+    # pyenv shell is not set
+    if [[ "$PYENV_SHELL" == *"pyenv: no shell-specific version configured"* ]]; then
```

Comment:
Is the above error message locale invariant? If not, you'd probably need to specify `LANG=C` as `$(LANG=C pyenv ...)`.

Reply:
As far as I can tell, it isn't localized, both strings seem hardcoded: https://github.com/pyenv/pyenv/blob/master/libexec/pyenv-sh-shell and https://github.com/pyenv/pyenv/blob/master/libexec/pyenv-local

Reply:
Thank you for checking! Anyway, you seem to have switched to another approach, so never mind.